src/app/hooks/queries/useAccountBalance.ts

```diff
@@ -27,6 +30,58 @@ const useAccountBalance = () => {
   const [isProcessingQueue, setIsProcessingQueue] = useState(false);
   const [queueLength, setQueueLength] = useState(0);
 
+  const fetchBrcCoinsBalances = async (ordinalsAddress: string) => {
```

Comment:
nit: we should have an util to handle this here and in useAccountBalance hook.

Reply:
agreed 👌 I added a todo comment